src/tests/ftest/control/dmg_server_set_logmasks.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2022 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+
+from control_test_base import ControlTestBase
+
+class DmgServerSetLogmasksTest(ControlTestBase):
+    # pylint: disable=too-many-ancestors
+    """Test Class Description:
+    Verify the server set-logmasks function of the dmg tool.
+    :avocado: recursive
+    """
+
+    def test_dmg_server_set_logmasks(self):
+        """
+        JIRA ID: DAOS-10900
+        Test Description: Test that server set-logmasks command completes successfully.
+        :avocado: tags=all,daily_regression
```

Comment:
no pr tag? Are you sure that the test is actually being run when this PR is being tested?

Reply:
From CI run #9: `https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-9471/9/artifact/Functional%20on%20EL%208/control/dmg_server_set_logmasks.py/job.log/*view*/` the command is run successfully:
```
2022-06-25 12:03:15,698 process          L0604 INFO | Running '/usr/bin/dmg -o /etc/daos/daos_control.yml -d -j server set-logmasks'
2022-06-25 12:03:15,832 process          L0416 DEBUG| [stderr] DEBUG 12:03:15.831771 main.go:216: debug output enabled
2022-06-25 12:03:15,832 process          L0416 DEBUG| [stderr] DEBUG 12:03:15.832617 main.go:244: control config loaded from /etc/daos/daos_control.yml
2022-06-25 12:03:15,834 process          L0416 DEBUG| [stderr] DEBUG 12:03:15.834457 server.go:54: set log masks request: &{unaryRequest:{request:{timeout:0 deadline:{wall:0 ext:0 loc:<nil>} Sys: HostList:[]} rpc:<nil>} Masks:}
2022-06-25 12:03:15,835 process          L0416 DEBUG| [stderr] DEBUG 12:03:15.834978 server.go:52: DAOS set engine log masks request: sys:"daos_server-2.3.100"
2022-06-25 12:03:15,835 process          L0416 DEBUG| [stderr] DEBUG 12:03:15.835642 rpc.go:254: request hosts: [wolf-101vm2:10001 wolf-101vm3:10001 wolf-101vm4:10001]
2022-06-25 12:03:15,890 process          L0416 DEBUG| [stderr] DEBUG 12:03:15.890228 server.go:69: DAOS set engine log masks response: &{HostErrorsResp:{HostErrors:map[]}}
2022-06-25 12:03:15,890 process          L0416 DEBUG| [stderr] DEBUG 12:03:15.890307 server.go:61: set log masks response: &{HostErrorsResp:{HostErrors:map[]}}
2022-06-25 12:03:15,890 process          L0416 DEBUG| [stdout] {
2022-06-25 12:03:15,890 process          L0416 DEBUG| [stdout]   "response": {
2022-06-25 12:03:15,890 process          L0416 DEBUG| [stdout]     "host_errors": {}
2022-06-25 12:03:15,891 process          L0416 DEBUG| [stdout]   },
2022-06-25 12:03:15,891 process          L0416 DEBUG| [stdout]   "error": null,
2022-06-25 12:03:15,891 process          L0416 DEBUG| [stdout]   "status": 0
2022-06-25 12:03:15,891 process          L0416 DEBUG| [stdout] }
2022-06-25 12:03:15,906 process          L0686 INFO | Command '/usr/bin/dmg -o /etc/daos/daos_control.yml -d -j server set-logmasks' finished with 0 after 0.2049553394317627s
```